resources/assets/lib/follow-user-mapping-button.tsx

```diff
@@ -43,29 +53,40 @@ export default class FollowUserMappingButton extends React.Component<Props, Stat
   }
 
   render() {
-    if (currentUser.id == null || currentUser.id === this.props.userId) {
+    if (currentUser.id === this.props.userId && !this.props.alwaysVisible) {
       return null;
     }
 
-    const title = osu.trans(`follows.mapping.${this.state.follow ? 'to_0' : 'to_1'}`);
+    const currentUserProfile = currentUser.id === this.props.userId;
+    const notLoggedIn = currentUser.id == null;
```

Comment:
maybe just combine an `||` of these as `canToggle` and use it where needed (always visible check above, title and disabled state below)